ga4gh/exceptions.py

```diff
@@ -6,11 +6,25 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import sys
 import zlib
+import inspect
 
 import ga4gh.protocol as protocol
 
 
+def getExceptionClass(errorCode):
```

Comment:
If this is only used for debugging purposes, why not put it in `convert_error_code.py`?